package.json

```diff
@@ -35,6 +35,7 @@
   },
   "devDependencies": {
     "ava": "^0.25.0",
+    "lodash": "^4.17.11",
```

Comment:
Does it needed only for test?

Reply:
yeah I required it only in test, it would not be installed when you `"npm install fx"`